frameworks/solid/project/base/src/components/Header.tsx.ejs

```diff
@@ -6,6 +6,7 @@ const icons = new Set([
   "Menu",
   "X",
   "House",
+  "Home",
```

Comment:
the "Home" has been renamed "House" in lucide-solid

Reply:
are you sure? When i tested it was complaining about Home being missing

Reply:
<img width="247" height="124" alt="Screenshot 2025-10-25 at 19 51 51" src="https://github.com/user-attachments/assets/da28f849-4888-4ae2-979e-e3adfd71cc08" />

Reply:
then we need to change the place its actually used too

Reply:
<img width="534" height="274" alt="Screenshot 2025-10-25 at 19 53 50" src="https://github.com/user-attachments/assets/dbf3f4d0-b058-457c-8a59-f79c5ac40e7f" />